2022-11-following-strategies-in-astrolab.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Following strategies in Astrolab
+subtitle: You can now follow trading strategies of the community
+slug: following-strategies-in-astrolab
+tags: cryptocurrency, trading, strategy, exchange, blog, astrolab
+cover: https://raw.githubusercontent.com/Drakkar-Software/OctoBot-Blog/master/resources/images/following-strategies-in-astrolab/cover.jpg
+domain: blog.octobot.online
+--- 
+
+
+# Follow the best strategies
+On Astrolab, you can subscribe to trading strategies. Subscribing to a strategy allows you to easily trade using a strategy made by someone else from the OctoBot community.
+
+When subscribed to a strategy, you can chose to use the strategy profile from your OctoBot. When you do so, your OctoBot will follow the strategy by coping any trade made by this strategy. Order amounts will be adapted to your current portfolio.
+
+# How to use a followed strategy ?
+1. Login on [Astrolab](https://www.astrolab.cloud/) and go to the desired strategy page 
+2. Click `Subscribe`
+![Following-strategies-pre-sub](https://raw.githubusercontent.com/Drakkar-Software/OctoBot-Blog/master/resources/images/following-strategies-in-astrolab/pre-sub.png)
+3. Now that you are subscribing to the profile, click `Copy download url`
+4. From your OctoBot, click `Import a profile`
```

Comment:
Should we precise that the user have to login to the community tab before copying?